src/pages/SupplierInvoicePage.js

```diff
@@ -111,12 +113,14 @@ export class SupplierInvoicePage extends React.Component {
 
   componentDidMount = () => {
     const { transaction, database } = this.props;
-    this.defaultFridge = database.objects('Location').find(location => location.isFridge);
+    this.FRIDGES = database.objects('Location').filter(({ isFridge }) => isFridge);
+    [this.DEFAULT_FRIDGE = null] = this.FRIDGES;
```

Comment:
this.FRIDGES is an array, check length and this.FRIDGES[0]

Reply:
hm, if the length is 0, it will assign `undefined`, which is then overwritten by the default `null`. And could only be a `Location` at this point

Reply:
ahh true, nice one